lib/compute/agent-nodes.ts

```diff
@@ -83,6 +83,7 @@ export class AgentNodes {
         numExecutors: 1,
         amiId: 'ami-0f6ceb3b3687a3fba',
         initScript: 'sudo apt-mark hold docker docker.io openssh-server && docker ps &&'
+        + ' (sudo killall -9 apt-get apt 2>&1 || echo) &&'
```

Comment:
I already explained it in the description this is a race condition since as soon as Ubuntu start up it will trigger auto upgrade, while we also trigger a apt to install certain updates, this race can cause an error in return and not able to start the runner correctly.